README.md

```diff
@@ -76,16 +82,16 @@ When working on this goal, you’ll encounter questions like:
 
 ### Clear and useful README
 
- - [ ] Repository includes a README file with installation and setup instructions. [25 points]
+ - [x] Repository includes a README file with installation and setup instructions. [25 points]
```

Comment:
Still can't get it to work but I combed through the code. You have a solid grasp on 'this'. I did load hugemanatee into my Postico and saw that the database was populated. OH, I love the spinning manatee, and am assuming that it's a REACT thing vs. a CSS transition.